tests/sample_data/multimodel_statistics/test_multimodel.py

```diff
@@ -396,3 +421,70 @@ def test_multimodel_0d_different_time_dimensions(timeseries_cubes_month):
     msg = "Tried to align cubes in multi-model statistics, but failed for cube"
     with pytest.raises(ValueError, match=msg):
         multimodel_test(cubes, span=span, statistic='mean')
+
+
+@pytest.mark.use_sample_data
+@pytest.mark.parametrize('span', SPAN_PARAMS)
+def test_multimodel_diff_scalar_time_ignore(timeseries_cubes_month, span):
+    """Ignore different scalar time dimensions.
+
+    Also remove air_pressure dimensions since this slightly differs across
+    cubes. See https://github.com/ESMValGroup/ESMValCore/issues/956.
```

Comment:
As far as I can see from the code and from running a test, the `air_pressure` coordinate is not removed, so I'm a bit confused by this comment. Could you explain?

Reply:
If you use `timeseries_cubes_month` without any changes, the test will (rightfully) [fail](https://github.com/ESMValGroup/ESMValCore/blob/dc09915a48a000b011e1ce14e396541d0d70eaa6/tests/sample_data/multimodel_statistics/test_multimodel.py#L201-L216) due to different `air_pressure` coordinates:

```py
cube 0 [100000.  92500.]
cube 1 [100000.  92500.]
...
cube 12 [100000.00000001  92500.00000001]
cube 13 [100000.00000001  92500.00000001]
cube 14 [100000.  92500.]
...

```

I am not entirely sure why that happens, but it's probably described in https://github.com/ESMValGroup/ESMValCore/issues/956.

Thus, in this test (and many others), the `air_pressure` coordinate is collapsed by using `cube[0, 0]`. Note that different coordinate `points` are fine for scalar coordinates, just not different coordinate metadata.

I will adapt the comment to make it clear that the coordinate is collapsed, not removed.

Reply:
Thanks! Now that we have the `esmvalcore.dataset.Dataset` class, we could use that to load the data and that should avoid the issue altogether. But that's something for another pull request.